admin/src/common/blocks/RichTextBlock.tsx

```diff
@@ -1,5 +1,23 @@
 import { createRichTextBlock } from "@comet/cms-admin";
+import { Typography } from "@mui/material";
+import { FormattedMessage } from "react-intl";
 
 import { LinkBlock } from "./LinkBlock";
 
-export const RichTextBlock = createRichTextBlock({ link: LinkBlock });
+export const RichTextBlock = createRichTextBlock({
+    link: LinkBlock,
+    rte: {
+        standardBlockType: "paragraph-standard",
+        blocktypeMap: {
+            "paragraph-standard": {
+                label: <FormattedMessage id="richTextBlock.paragraph-one" defaultMessage="Paragraph Standard" />,
+            },
+            "paragraph-small": {
+                label: <FormattedMessage id="richTextBlock.paragraph-two" defaultMessage="Paragraph Small" />,
```

Comment:
The IDs are incorrect and should be camelCase. 

```suggestion
                label: <FormattedMessage id="richTextBlock.paragraphStandard" defaultMessage="Paragraph Standard" />,
            },
            "paragraph-small": {
                label: <FormattedMessage id="richTextBlock.paragraphSmall" defaultMessage="Paragraph Small" />,
```